source/documentation/breaking-changes/import.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Breaking Change: `@import` and global built-in functions'
+introduction: >
+  Originally, Sass used `@import` rules to load other files with a single
+  global namespace, with all built-in functions also available globally. We're
+  now deprecating both Sass `@import` rules and global built-in functions, now
+  that the module system (`@use` and `@forward` rules) has been available for
+  several years.
+---
+
+Sass historically loaded dependencies in a single global namespace using
+`@import` rules. This causes numerous problems, requiring Sass members to be
+manually namespaced to avoid conflicts, slowing down compilation when the same
+file is imported more than once, and making it very difficult for both humans
+and tools to tell where a given variable, mixin, or function comes from.
+
+The module system fixes these problems and brings Sass's modularity with the
+best practices of other modern languages, but we can't get the full benefits
```

Comment:
Is this supposed to be "brings Sass's modularity **up to par** with the best practices of other modern languages"?